plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -839,6 +839,7 @@ fu_dell_dock_ec_write_fw (FuDevice *device,
 		}
 	}
 
+	fu_device_set_status (device, FWUPD_STATUS_DEVICE_RESTART);
```

Comment:
likewise Actually the device doesn't restart at the end, it restarts when you unplug it.